packages/playground/blueprints/src/lib/compile.ts

```diff
@@ -220,6 +221,9 @@ export function compileBlueprint(
 			networking: blueprint.features?.networking ?? false,
 		},
 		run: async (playground: UniversalPHP) => {
+			// Initialize logger
+			const logger = get_logger();
+			logger.addPlaygroundRequestEndListener(playground);
```

Comment:
Note this code is executed in the website context, not in the remote.html context. As you noted in one of the comments, the `remote` package could listen to those events. More specifically, `worker-thread.ts`, where WebPHP is initialized, could read WordPress error logs and send them to `remote.html`. Then, `remote.html` could collect those plus any JS errors, and send them over to the website. Then, website could collect those plus any website-specific JS errors, and use all of that information – e.g. buffer it for the "Report a problem" modal.

Reply:
Makes sense! If that's ok I would like to merge this part as-is and work on the JS logger in a separate PR.
The current implementation is mostly a proof of concept.